types/types.go

```diff
@@ -542,7 +542,10 @@ type SystemContext struct {
 	// Allow contacting docker registries over HTTP, or HTTPS with failed TLS verification. Note that this does not affect other TLS connections.
 	DockerInsecureSkipTLSVerify OptionalBool
 	// if nil, the library tries to parse ~/.docker/config.json to retrieve credentials
+	// Ignored if DockerRegistryToken is non-empty.
 	DockerAuthConfig *DockerAuthConfig
+	// if not "", the library uses this registry token to authenticate to the registry
+	DockerRegistryToken string
```

Comment:
Maybe `DockerRegistryBearerToken`, to differentiate this from the “identity token” discussed elsewhere.

Reply:
`moby/moby` says `RegistryToken` and `IdentityToken`, so I feel `DockerRegistryToken` and `DockerIdentityToken` is natural in `containers/image`.
https://github.com/moby/moby/blob/8e610b2b55bfd1bfa9436ab110d311f5e8a74dcb/api/types/auth.go#L16-L21

But I can follow you if you still want me to change it.

Reply:
I’d weakly prefer the “bearer” to be in there, but it’s not blocking for me.

Do document it at least in the comment, please. “Registry token” is ambiguous without extra context, all tokens are “for a registry”.

Reply:
I got it. I fixed it.
https://github.com/containers/image/pull/842/commits/ed44669ebe2fc54a45772e965ab68b788bed4935